docs/02-app/01-building-your-application/01-routing/11-middleware.mdx

```diff
@@ -76,7 +76,7 @@ export const config = {
      * - _next/image (image optimization files)
      * - favicon.ico (favicon file)
      */
-    '/((?!api|_next/static|_next/image|favicon.ico).*)',
+    '/((?!api|_next/static|_next/image|favicon.ico).*)?',
```

Comment:
Hey @hydRAnger, thanks for your suggestion! 

I think the addition of `?` could end up confusing people as it makes the preceding group optional. 

Although complex, you could try `/^((?!api|_next\/static|_next\/image|favicon.ico).*|\/)$/` to match the root. Let's see if that works for your use case.